hide/comp/cdb/Cell.hx

```diff
@@ -147,6 +148,14 @@ class Cell extends Component {
 		updateClasses();
 	}
 
+	function watchFile( file : String ) {
+        if( file == null ) return;  // array.index of

```

Comment:
Hide/heaps use tabs as indentation, please avoid mixing both

Reply:
I really don't know how spaces got in there...
fixed now